### ISSUE TITLE
ClinicalBERT for Heart failure patients

### ISSUE THREAD

**Issue by johagg17** [kexinhuang12345/clinicalBERT#17]
Hello! 

First, I want to thank you for the provided code and weights for ClinicalBERT. 

I am trying to use the ClinicalBERT for predicting if a patient will be diagnosed with heart failure or not. 
You provided three specific models, one for discharge readmission, one for early readmission, and one for pretraining.

Am I misunderstanding the use of clinicalBERT? Is it only specified for readmission tasks or can I use it for my use-case (heart failure)? 
If yes, should I use the model in the "pretraining" folder, or could I use the discharge model (if now I am finetuning on discharge summaries)? 

To summarize what I just said, 
I want to use your model, and then finetune it for heart failure patients, which pre-trained model should I use?

Once again, thank you. :) 

Kind regards, 
Johannes

**Comment by kexinhuang12345** [comment]
Hi! Currently, it is only for readimssion case. But if you have a dataset with clinical notes and heart failure label, then you can use pretrained checkpoint and fine-tune with your new dataset!
